lint/src/test/java/net/grandcentrix/thirtyinch/MissingViewInThirtyInchDetectorTest.java

```diff
@@ -0,0 +1,261 @@
+package net.grandcentrix.thirtyinch;
+
+import com.android.tools.lint.checks.infrastructure.LintDetectorTest;
+import com.android.tools.lint.detector.api.Detector;
+import com.android.tools.lint.detector.api.Issue;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MissingViewInThirtyInchDetectorTest extends LintDetectorTest {
+
+    private static final String NO_WARNINGS = "No warnings.";
+
+    /* Stubbed-out source files */
+
+    private final TestFile tiActivityStub = java("" +
+            "package net.grandcentrix.thirtyinch;\n" +
+            "public abstract class TiActivity<P extends TiPresenter<V>, V extends TiView> {\n" +
+            "}");
+
+    private final TestFile tiFragmentStub = java("" +
+            "package net.grandcentrix.thirtyinch;\n" +
+            "public abstract class TiFragment<P extends TiPresenter<V>, V extends TiView> {\n" +
+            "}");
+
+    private final TestFile tiPresenterStub = java("" +
+            "package net.grandcentrix.thirtyinch;\n" +
+            "public abstract class TiPresenter<V extends TiView> {\n" +
+            "}");
+
+    private final TestFile tiViewStub = java("" +
+            "package net.grandcentrix.thirtyinch;\n" +
+            "public interface TiView {\n" +
+            "}");
+
+//    private final TestFile caBasePluginStub = java("" +
```

Comment:
In the process of separating Lint detection around the different available use cases, these were left behind.